src/meltano/core/plugin/dbt/base.py

```diff
@@ -15,8 +16,12 @@
 
 
 class DbtInvoker(PluginInvoker):
-    def Popen_options(self):
-        return {**super().Popen_options(), "cwd": self.plugin_config["project_dir"]}
+    def cwd(
+        self, command: Optional[str] = None, env: Optional[dict] = None
+    ) -> Optional[Path]:
+        return (
+            super().cwd(command=command, env=env) or self.plugin_config["project_dir"]
+        )
```

Comment:
Superceding my prior comment about `cwd` replacing the `project_dir` setting of dbt, I see here that yes, absolutely this appears to replace that functionality - and I like that we're doing it in a generic way for all plugins, which would not require special per-plugin logic internally within Meltano.

Also - Thanks for keeping this backwards compatible. As much as I like the generic approach, we also don't want this to break existing dbt capabilities/behaviors.